.github/actions/README.md

```diff
@@ -18,19 +18,19 @@ and is difficult to get right.
     
 In particular, *composite actions* need to use inline scripts and cannot invoke script files stored in the builder's repository.
```

Comment:
```suggestion
In particular, *composite actions* need should not invoke script files stored in the git repository. Only inline scripts are permitted under these rules.
```

Reply:
"*composite actions* need should not invoke script" -> "*composite actions* should not invoke script" ?